Sources/SwiftInspectorVisitors/Tests/ExtensionVisitorSpec.swift

```diff
@@ -127,18 +127,21 @@ final class ExtensionVisitorSpec: QuickSpec {
                   struct InnerStruct {}
                   class InnerClass {}
                   enum InnerEnum {}
+                  typealias InnerTypealias = Void
```

Comment:
I think this context block is off?
<img width="596" alt="Screen Shot 2021-04-05 at 8 36 04 PM" src="https://user-images.githubusercontent.com/1791049/113655398-910b8980-964e-11eb-921f-faf9a90f2a5d.png">

Reply:
ahh shoot good call. Deleted the inner context call 77bdc92e6b7f0f2647420f09e5f33c0ec1c10ddd